docs/source/accelerators/gpu.rst

```diff
@@ -4,10 +4,29 @@
     from pytorch_lightning.trainer.trainer import Trainer
     from pytorch_lightning.core.lightning import LightningModule
 
-.. _multi_gpu:
+.. _gpu:
+
+Graphics Processing Unit (GPU)
+==============================
+
+
+Single GPU Training
+-------------------
+
+Make sure you're running on a machine with at least one GPU. There's no need to specify any NVIDIA flags
+as Lightning will do it for you.
+
+.. testcode::
+    :skipif: torch.cuda.device_count() < 1
+
+    # train on 1 GPU (using dp mode)
+    trainer = Trainer(gpus=1)
```

Comment:
```suggestion
    # train on 1 GPU (using DP mode)
    trainer = Trainer(gpus=1)
    # or
    trainer = Trainer(devices=1, accelerator='gpu')
```
default is DP?

Reply:
Updated! And I don't think we should confuse users with new ways for now